hydromt_fiat/fiat.py

```diff
@@ -313,13 +312,15 @@ def setup_exposure_buildings(
         max_potential_damage: Union[str, Path],
         ground_floor_height: Union[int, float, str, Path, None],
         unit: str,
```

Comment:
this should be Units = None, and maybe put it somewhere on top, because now it is in between the ground_floor_height and the gfh_unit which is kind of confusing.

Moreover maybe better to rename it to length_unit as we discussed?

Reply:
If I make it None I cannot place it above a parameter without default input. Also I'm wondering if the model unit maybe should not be optional? It seems important to set the base unit by the user and not a default that the user may not be aware of? What's your opinion on that?